src/styles.scss

```diff
@@ -836,14 +843,21 @@ $green-font-color: #19B028;
   font-size: 15px !important;
 }
 
+// Changed Dropdown Css
 .change-account-selector {
-  margin-left: -10px;
-  position: relative;
-  z-index: 30;
-  background-color: var(--secondary);
-  width: 185px;
-  padding-top: 8px;
-  padding-bottom: 8px;
+	margin-left: -10px;
+	position: relative;
+	z-index: 30;
+	background-color: white;
```

Comment:
this looks a bit strange on non-light themes:
![image](https://user-images.githubusercontent.com/81658138/124681911-6fed4b80-de7e-11eb-8137-50ec41f5b775.png)
```suggestion
	background-color: var(--secondary);
```